requirements.txt

```diff
@@ -0,0 +1,6 @@
+rich~=12.4.4
+requests~=2.28.0
+bonbast~=0.0.5
+setuptools~=60.2.0
+bs4~=0.0.1
+beautifulsoup4~=4.11.1
```

Comment:
We shouldn't add this one here.
Delete requirements and add this to pyproject.toml